trade-finance-manager-api/src/v1/services/deal-cancellation/deal-cancellation.service.ts

```diff
@@ -0,0 +1,64 @@
+import { endOfDay, format, isAfter, toDate } from 'date-fns';
+import { DATE_FORMATS, DEAL_TYPE, TEAMS, TfmDealCancellation } from '@ukef/dtfs2-common';
+import sendTfmEmail from '../send-tfm-email';
+import { CANCEL_DEAL_PAST_DATE, CANCEL_DEAL_FUTURE_DATE } from '../../../constants/email-template-ids';
+import * as api from '../../api';
+import { formatFacilityIds } from './helpers/format-facility-ids';
+import { UKEF_ID } from '../../../constants/deals';
+
+export class DealCancellationService {
+  /**
+   * Sends email to PIM email address confirming the cancellation has been submitted
+   * @param ukefDealId ukef deal id
+   * @param dealCancellation deal cancellation object
+   * @param facilityIds ukef facility ids
+   * @private - This function is exported for unit testing only and should not be used outside of the service
+   */
+  public static async sendDealCancellationEmail(ukefDealId: string, dealCancellation: TfmDealCancellation, facilityIds: string[]): Promise<void> {
```

Comment:
What would the try catch contain? We need the errors to bubble up to `trade-finance-manager-api\src\v1\controllers\deal-cancellation\submit-deal-cancellation.controller.ts` in order to send the api responses accordingly.

I dont think adding the following would be very helpful, especially since it would hide the location of the original error when its logged a second time in the controller.
```
try {

} catch (e){
  console.error('an error', e)
  throw e
}
```

Reply:
I was thinking it could contain a `console.info` and throw an error in the catch. Makes sense to let the error bubble up.

Can we add a `console.info` at the top?

Reply:
Bump, can we add a `console.info`?